nodezator/fontsman/cache.py

```diff
@@ -118,6 +119,16 @@ def get_font(font_path, desired_height):
     surpass the desired height, that is, the font with
     surfaces of height 35.
     """
+    def raise_if_unreachable(desired, attempted):
+        """Detect when technical limitations of font interpolation
+        will prevent creation of certain font sizes.
+        """
+        sizes = [desired, attempted]
+
+        if True in list(map(lambda s: s < 0 or s >= 65536, sizes)):
```

Comment:
🤓 

might not be efficient. is called infrequently. thus, i hope it's okay to do this?

Reply:
```suggestion
        if any(map(lambda s: s < 0 or s >= 65536, sizes)):
```

Reply:
This is just slightly quicker since it doesn't build a list, but more elegant and Pythonic.

Also, I don't like to change people's contributions without their permission, but for such small change I'll take the liberty to commit it here from GitHub's web interface. Please, let me know if I overstepped here.